CoffeeChat/Controller/DemographicsViewController.swift

```diff
@@ -36,125 +47,109 @@ class DemographicsViewController: UIViewController {
     }
 
     override func viewDidLoad() {
-        view.backgroundColor = .white
+        view.backgroundColor = .backgroundLightGreen
+        navigationController?.navigationBar.isHidden = true
 
         helloLabel.text = "Hi \(userDefaults.string(forKey: "userFirstName") ?? "user")!"
         helloLabel.textColor = .textBlack
-        helloLabel.font = .systemFont(ofSize: 24, weight: .bold)
+        helloLabel.font = ._24CircularStdMedium
         view.addSubview(helloLabel)
 
         greetingLabel.text = "Let's get to know you better."
         greetingLabel.textColor = .textBlack
-        greetingLabel.font = .systemFont(ofSize: 24, weight: .bold)
+        greetingLabel.font = ._24CircularStdMedium
         view.addSubview(greetingLabel)
 
-        classTextField.backgroundColor = .backgroundLightGray
-        classTextField.attributedPlaceholder = NSAttributedString(string: "Class of...", attributes: [NSAttributedString.Key.foregroundColor: UIColor.textDarkGray])
-        classTextField.keyboardType = .decimalPad
-        classTextField.leftView = UIView(frame: CGRect(x: 0, y: 0, width: 12, height: 49))
-        classTextField.leftViewMode = .always
-        classTextField.layer.cornerRadius = fieldsCornerRadius
-        view.addSubview(classTextField)
-
-        majorSearchBar.backgroundColor = .backgroundLightGray
-        majorSearchBar.backgroundImage = UIImage()
-        // majorSearchBar.delegate = self
-        if let majorTextField = majorSearchBar.value(forKey: "searchField") as? UITextField {
-            majorTextField.backgroundColor = .backgroundLightGray
-            majorTextField.attributedPlaceholder = NSAttributedString(string: "Major", attributes: [NSAttributedString.Key.foregroundColor: UIColor.textDarkGray])
-            majorTextField.clearButtonMode = .never
-        }
-        majorSearchBar.setImage(UIImage(), for: .search, state: .normal)
-        majorSearchBar.layer.cornerRadius = fieldsCornerRadius
-        majorSearchBar.searchTextPositionAdjustment = UIOffset(horizontal: -6, vertical: 0)
-        majorSearchBar.showsCancelButton = false
-        view.addSubview(majorSearchBar)
-
-        hometownSearchBar.backgroundColor = .backgroundLightGray
-        hometownSearchBar.backgroundImage = UIImage()
-        // hometownSearchBar.delegate = self
-        if let hometownSearchField = hometownSearchBar.value(forKey: "searchField") as? UITextField {
-            hometownSearchField.backgroundColor = .backgroundLightGray
-            hometownSearchField.attributedPlaceholder = NSAttributedString(
-                string: "Hometown",
-                attributes: [NSAttributedString.Key.foregroundColor: UIColor.textDarkGray]
-            )
-            hometownSearchField.clearButtonMode = .never
+        // Renders the valid graduation years based on current year.
+        let currentYear = Calendar.current.component(.year, from: Date())
+        for year in currentYear...currentYear+4 {
```

Comment:
can we put `currentYear + 4` into a `let` before w/ a name describing why its four